src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,168 @@
+                System.out.println("Now you have " + noOfTask + " tasks in the list.");
+            }
+        }
+        return noOfTask;
+    }
+
+    private static int Todo(String line, Task[] list, int noOfTask, int divider) {
+
+        //Run description error
+        if (line.equals("todo") || line.equals("todo ")) {
+            new Exception("description", "todo");
+
+        //If no error, run code to store to do
+        } else {
+            list[noOfTask] = new Todo(line.substring(divider + 1));
+            noOfTask++;
+            System.out.println("Got it. I've added this task:");
+            System.out.println("  " + list[noOfTask - 1]);
+            if ((noOfTask - 1) == 0) {
+                System.out.println("Now you have " + noOfTask + " task in the list.");
+            } else {
+                System.out.println("Now you have " + noOfTask + " tasks in the list.");
+            }
+        }
+        return noOfTask;
+    }
+
+    private static int Deadline(String line, Task[] list, int noOfTask, int divider) {
+        int dividerDate = line.indexOf("/");
+
+        //Run description error
+        if (line.equals("deadline") || line.equals("deadline ")) {
+            new Exception("description", "deadline");
+
+        //Run date error
+        } else if (line.endsWith("/") || line.endsWith("/ ")) {
+            new Exception("date", "deadline");
+
+        //If no error, run code to store deadline
+        } else {
+            list[noOfTask] = new Deadline(line.substring((divider + 1),
+                    (dividerDate - 1)), line.substring(dividerDate + 4));
+            noOfTask++;
+            System.out.println("Got it. I've added this task:");
+            System.out.println("  " + list[noOfTask - 1]);
+            if ((noOfTask - 1) <= 0) {
+                System.out.println("Now you have " + noOfTask + " task in the list.");
+            } else {
+                System.out.println("Now you have " + noOfTask + " tasks in the list.");
+            }
+        }
+        return noOfTask;
+    }
+
+    private static void doneTask(String line, Task[] list, int divider) {
+
+        //Run description error
+        if (line.equals("done") || line.equals("done ")) {
+            new Exception("description", "done");
+
+        //If no description error, run code to tick done
+        } else {
+            try {
+                int taskNumber = Integer.parseInt(line.substring(divider + 1));
+                list[taskNumber - 1].setDone();
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println("  " + list[taskNumber - 1]);
+
+            //If no number is input, throw error
+            } catch (NumberFormatException e) {
+                System.out.println("Please input the task number!");
+            }
+        }
+    }
+
+    private static void listTask(Task[] list, int noOfTask) {
+
+        //Run empty error
+        if (noOfTask == 0) {
```

Comment:
You should try to avoid magic constants. You could consider using a named constant instead.